src/content/docs/extensions/iceberg.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: "iceberg extension"
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+## Usage
+
+The `iceberg` extension adds support for `iceberg` objects, currently including scanning from [Apache Iceberg format](https://iceberg.apache.org/).
+
+The iceberg functionality is not available by default, so you would first need to install the iceberg
+extension by running the following commands:
+
+```sql
+INSTALL iceberg;
+LOAD EXTENSION iceberg;
+```
+
+### Example dataset
+
+To test the examples, download the [iceberg_data.zip](TODO: upload iceberg sample data for download) file and unzip it.
+
+## Usage
+
+### Common Parameters
+
+```
+┌───────────────────────────────┬───────────┬──────────────┬──────────────────────────────────────────────────────────────────────────────┐
+│ Parameter                     │ Type      │ Default      │ Description                                                                  │
+├───────────────────────────────┼───────────┼──────────────┼──────────────────────────────────────────────────────────────────────────────┤
+│ allow_moved_paths             │ BOOLEAN   │ false        │ Allows scanning Iceberg tables that are moved                                │
+│ metadata_compression_codec    │ STRING    │ ''           │ Treats metadata files as when set to 'gzip'                                  │
+│ version                       │ STRING    │ '?'          │ Provides an explicit version string, hint file or guessing                   │
```

Comment:
What do you mean "hint file or guessing"?

Reply:
removed this unclear statement